From b048fc3161f5949220b19faf49437c715b64521a Mon Sep 17 00:00:00 2001
From: Vincent Abriou <vincent.abriou@st.com>
Date: Mon, 8 Apr 2019 12:08:16 +0200
Subject: [PATCH 4/4] TFLite pip package: fix python execution issue

No Module named 'interpreter'
'interpreter' does not refer to any module but
'tflite_runtime.lite.interpreter' does.

Signed-off-by: Vincent Abriou <vincent.abriou@st.com>
---
 tensorflow/lite/tools/pip_package/build_pip_package.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tensorflow/lite/tools/pip_package/build_pip_package.sh b/tensorflow/lite/tools/pip_package/build_pip_package.sh
index 5dc9ebf..e720a7a 100644
--- a/tensorflow/lite/tools/pip_package/build_pip_package.sh
+++ b/tensorflow/lite/tools/pip_package/build_pip_package.sh
@@ -35,7 +35,7 @@ import tflite_runtime.lite.interpreter
 EOF
 
 cat > $BUILD_ROOT/tflite_runtime/lite/__init__.py <<EOF;
-from interpreter import Interpreter as Interpreter
+from tflite_runtime.lite.interpreter import Interpreter as Interpreter
 EOF
 
 cat > $BUILD_ROOT/tflite_runtime/lite/python/__init__.py <<EOF;
-- 
2.7.4

